src/services/api/api_service.cpp

```diff
@@ -18,6 +18,44 @@ namespace big
 		g_api_service = nullptr;
 	}
 
+	std::string api_service::get_translation_from_LibreTranslate(std::string message, std::string target_language)
```

Comment:
Function name is a mix between snake_case and CamelCase.

2 options, change the name completely to snake_case or rework the name and add a comment.

```cpp
/**
 * Makes an API call to a LibreTranslate endpoint.
 */
std::string api_service::get_translation(std::string message, std::string target_language)
```